app/views/notes/index.html.erb

```diff
@@ -1,13 +1,28 @@
 <% notes.each_with_index do |note, i| %>
+
+
+
   <% if authorized?(note) %>
     <h3>
       Note <%= i + 1 %>
     </h3>
-    <div>Name: <%= note.note_with_type %></div>
-    <div>Author: <%= link_to note.user.name, note.user %></div>
-    <div>Category: <%= link_to note.category.name, note.category %></div>
-    <div>Body: <%= note.body %></div>
+
+    <div class="row">
+      <div class="card blue-grey file-card col s8 offset-s2">
+        <div class="card-content white-text">
+          <h3 class="card-title">Note <%= i + 1 %></h3>
+            <div>
+              <div>Name: <%= note.note_with_type %></div>
+              <div>Author: <%= note.user_id %></div>
```

Comment:
You removed a lot of the link_to's - please make sure users can get to the Author/user show, Category/category show, and Notes show pages.